godspeed/states/world.py

```diff
@@ -264,12 +266,52 @@ def update(self, event_info):
 
                 break
 
+    def handle_player_spike_collision(self):
+        for spike in self.spikes:
+            if self.player.collides(spike):
+                self.player.alive = False
+                break
+
+    def update(self, event_info):
+        super().update(event_info)
+        if self.player.alive:
+            self.player.handle_input(event_info["events"])
+
+        # Handle Player collision with other entities
+        self.player.vel = self.player.VEL
+        self.handle_player_platform_collision(event_info["dt"])
+        self.handle_player_spike_collision()
+
+        if not self.player.alive:
+            godspeed.common.UNIVERSAL_SPEEDUP = -(
```

Comment:
Thought this was a constant.
It should not be uppercase if its value can change over time.